data-access/bestuursperiode.ts

```diff
@@ -0,0 +1,61 @@
+import { sparqlEscapeString } from 'mu';
+import { querySudo } from '@lblod/mu-auth-sudo';
+import {
+  findFirstSparqlResult,
+  getBooleanSparqlResult,
+} from '../util/sparql-result';
+import { HttpError } from '../util/http-error';
+import { STATUS_CODE } from '../util/constants';
+
+export const bestuursperiode = {
+  isExisting,
+  getIdForUri,
+};
+
+async function isExisting(bestuursperiodeId: string): Promise<boolean> {
+  const askIfExists = `
+    PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+    PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+      ASK {
+        GRAPH ?bestuursperiodeGraph {
+          ?bestuursperiode a ext:Bestuursperiode;
+            mu:uuid ${sparqlEscapeString(bestuursperiodeId)}.
+        }
+
+        FILTER ( ?bestuursperiodeGraph != <http://mu.semte.ch/vocabularies/ext/FormHistory> )
+      }
+    `;
+
+  const result = await querySudo(askIfExists);
+
+  return getBooleanSparqlResult(result);
+}
+
+async function getIdForUri(bestuursperiodeUri: string): Promise<string> {
```

Comment:
This can be removed if we get the ember model depending on the URI in the repositories in the frontend*

Reply:
dont' think you need it, would remove it, either you do it in the getBestuursperiode query or you refactor it to do it when you update the current fractie.